### PR TITLE
[riverpod_generator] `dev` - Add `provider_name_prefix` and `provider_family_name_prefix` to `build.yaml`

### PR DESCRIPTION
**Pull request by ValentinVignal**
## Related Issues

Related to https://github.com/rrousselGit/riverpod/issues/3189 and follow up of https://github.com/rrousselGit/riverpod/pull/3348

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).

- [x] I have updated the `CHANGELOG.md` of the relevant packages.
      Changelog files must be edited under the form:

  ```md
  ## Unreleased fix/major/minor

  - Description of your change. (thanks to @yourGithubId)
  ```

- [x] If this contains new features or behavior changes,
      I have updated the documentation to match those changes.

### CODE REVIEW

File: packages/riverpod_generator/CHANGELOG.md [hunk 1]
```diff
@@ -98,6 +98,10 @@ This comes with a few minor restrictions:
 - **Breaking**: Arguments of the form `fn(void myParameter())`
   are no-longer supported. Instead use `fn(void Function() myParameter)`.
 
+## Unreleased minor
+
+- Adds `provider_name_prefix` and `provider_family_name_prefix` to `build.yaml`. (thanks to @ValentinVignal)
+
```
Comment (ValentinVignal) [inline packages/riverpod_generator/CHANGELOG.md:104]: I'm not too sure this is where you would want the changelog to be

**Comment by ValentinVignal** [comment]
@rrousselGit The test "[riverpod_lint (master, packages/riverpod_analyzer_utils_tests)](https://github.com/rrousselGit/riverpod/actions/runs/7992000378/job/21824521360?pr=3355#logs)" also fails for me on my local on `dev`, do you want me to do something to fix them ? I might need some help on that

**Comment by rrousselGit** [comment]
Don't worry about it. I'm currently working on it.

Your PR looks good! I'll merge this and the master PR once I'm done fixing dev

**Comment by rrousselGit** [comment]
Thanks again!

**Pull request merged**